lib/vintage_net_mobile/signal_monitor.ex

```diff
@@ -62,7 +62,7 @@ defmodule VintageNetMobile.SignalMonitor do
     if connected?(state) do
       # Only poll if connected, since some modems don't like it when they're not connected
 
-      # Spec says AT+CSQ max response time is 300 ms.
+      # Spec says AT+CSQ max response time is 500 ms.
```

Comment:
I think this is an example of where the comment was kind of right - the Quectel BG96 spec said 300 ms. 3GPP TS 27.007 doesn't give a max response time.

```suggestion
      # The AT+CSQ response should be quick. Quectel specifies a max response of 300 ms for
      # the BG96 and EC25. Other modules should be similar.
```